app/views/comments/_comment_item.html.haml

```diff
@@ -1,20 +1,29 @@
-%figure.comment-item.panel{id: "comment#{comment.id}"}
-  - if with_address
-    %h2.comment-address.panel-heading
-      In
-      %strong #{comment.application.suburb} #{comment.application.state}
-      on
+- if with_address
+  %h2.comment-location
+    In
+    %strong #{comment.application.suburb} #{comment.application.state}
+    on
+    = link_to application_path(comment.application.id), title: "View application", class: "comment-application" do
       = display_description_with_address(comment.application)
+%figure.comment-item.panel{id: "comment#{comment.id}"}
   .panel-body
-    %figcaption.comment-meta
+    %figcaption.comment-meta.comment-heading
       %span.comment-author #{comment.name}
       %span.comment-time-block
         commented
-        %time.comment-time{datetime: comment.updated_at.strftime("%F")}
-          #{time_ago_in_words(comment.updated_at)} ago
+        = link_to comment_path(comment), title: "View comment" do
+          %time.comment-time{datetime: comment.updated_at.strftime("%F")}
+            #{time_ago_in_words(comment.updated_at)} ago
     %blockquote.comment-text= comment_as_html(comment.text)
-    .comment-actions
-      - if current_user && current_user.admin?
-        = link_to "admin", admin_application_comment_path(comment), class: 'comment-action'
-      - unless current_page?(new_comment_report_path(comment.id))
-        = link_to "report comment", new_comment_report_path(comment), title: "Report this comment by #{comment.name} for removal", class: "comment-action"
+    %footer
+      %p.comment-meta
+        - # TODO: Change this to show the date in user local time.
+        - #       Could be nice and human to do 'today' and 'yesterday'
+        = link_to "Posted #{ comment.updated_at.strftime("%e %b %Y") }", comment_path(comment), title: "View comment"
+        and
+        %span delivered to #{comment.application.authority.full_name}
```

Comment:
Instead of "delivered", how about "sent". It's a simpler word and it's more commonly used when talking about emails.